### DISCUSSION TITLE
How do i install TWRP on Redmi Note 9 Merlin

### DISCUSSION THREAD

**Discussion by Exannger** [community/community#44872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Ive tried to install several times TWRP recovery into my phone that is Redmi Note 9 normal one called Merlin. (Redmi Note 9 Merlin) and is not listed on the page. What should i choose then? 

@Teamwin

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.


Also, you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
